packages/lisk-transactions/src/transactions/helpers/verify_multisignatures.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { TransactionError } from '../../errors';
+import { TransactionJSON } from '../../transaction_types';
+import { verifySignature } from './verify_signature';
+
+interface VerifyReturn {
+	readonly verified: boolean;
+	readonly errors: ReadonlyArray<TransactionError>;
+}
+
+export const verifyMultisignatures = (
+	memberPublicKeys: ReadonlyArray<string> = [],
+	minimumValidations: number = 0,
```

Comment:
assigning minimum would be dangerous here. if we put undefined, i will always valid